change/@fluentui-react-portal-3d88ca5d-d80e-40b2-9cb7-8bb39a62da74.json

```
@@ -0,0 +1,7 @@
+{
+  "type": "prerelease",
+  "comment": "react-portal: Replacing use of functions in makeStyles with direct use of tokens.",
```

```suggestion
  "comment": "Replacing use of functions in makeStyles with direct use of tokens",
```

nit: We will have it under `react-portal` section anyway